python/DEV_SETUP.md

```diff
@@ -255,14 +255,14 @@ class A:
         self.c = c
 ```
 
-You can uses the `SKGenericModel` to convert these to pydantic serializable classes.
+You can uses the `KernelGenericModel` to convert these to pydantic serializable classes.
```

Comment:
"You can uses" -> "You can use"

Reply:
Actually you can completely delete all mentions of SKGenericModel because it is no longer used, just there for legacy and should be removed by v1 if not sooner

Reply:
Replaced it with `KernelBaseModel`